interactive_ai/workflows/project_ie/job/repos/binary_storage_repo.py

```diff
@@ -72,13 +72,16 @@ def get_object_types() -> tuple[BinaryObjectType, ...]:
             if object_type not in BinaryStorageRepo.BLACKLISTED_OBJECT_TYPES
         )
 
-    def get_all_objects_by_type(self, object_type: BinaryObjectType, target_folder: str) -> Iterator[tuple[str, str]]:
+    def get_all_objects_by_type(
+        self, object_type: BinaryObjectType, target_folder: str, include_binary_paths: set[str] | None = None
```

Comment:
Rename suggestion: `include_binary_paths` -> `whitelisted_paths`

`include_binary_paths` sounds more like a boolean param to my ear